www/js/pages/profile/profile.template.html

```diff
@@ -316,7 +316,10 @@ <h1 class="page-title">
             <div class="row">
                 <div class="col center">
                     <span class="help-block error center" ng-if="vm.notAnyChanges">{{'financial-assessment.no_change' | translate}}</span>
-                    <button class="button button-block" ng-click="vm.updateProfile()" ng-disabled="!profileForm.$valid || vm.disableUpdateButton  || (vm.isFinancial && vm.profile.tax_residence  == null) ">
+                    <button class="button button-block"
```

Comment:
We have some cases in which the user has not set the country of residence at all and he/she gets a notification to choose the country and we redirect him/her to update profile page, we have to have update button enabled for that case